bigbluebutton-html5/imports/ui/components/nav-bar/service.js

```diff
@@ -2,6 +2,28 @@ import Auth from '/imports/ui/services/auth';
 import Breakouts from '/imports/api/breakouts';
 import { makeCall } from '/imports/ui/services/api';
 import Meetings from '/imports/api/meetings';
+import fp from 'lodash/fp';
+
+const getBreakoutByUserId = userId => Breakouts.find({ 'users.userId': userId }).fetch();
+
+const getBreakoutByUser = user => Breakouts.findOne({ users: user });
+
+const getUsersFromBreakouts = breakoutsArray => breakoutsArray
+  .map(breakout => breakout.users)
+  .flat();
```

Comment:
(2nd instance) @capilkey reported that `.flat()` is breaking the client on Edge`
@Tainan404 could you please use an alternative in a new PR?